scaffolds/nextjs-dedicated-wallet/template/README.md

```diff
@@ -2,6 +2,70 @@ This scaffold is meant to help you bootstrap your own projects with Magic's [Ded
 
 The folder structure of this scaffold is designed to encapsulate all things Magic into one place so you can easily add or remove components and functionality. For example, all Magic-specific components are in the `src/components/magic` directory while generic UI components are in the `src/components/ui` directory.
 
```

Comment:
Add a section that explains important directories and architecture. 
> This project uses Next.js but relies on fairly standard React components and hooks. Magic-related components are in the `/src/components/magic` directory, all other UI components are in the `/src/components/ui` directory, and hooks are in the `/src/hooks` directory. 
>
> ### How to modify this project 
> 
> ### Use this project as a reference for how to implement Magic in your own project
> Look at how we initialize Magic in the `MagicProvider` hook and utilize web3 stuff from `Web3.js` hook

..... alright this is rambly but we talked about this live.